x-pack/plugins/monitoring/server/kibana_monitoring/bulk_uploader.js

```diff
@@ -140,7 +140,7 @@ export class BulkUploader {
   async _fetchAndUpload(usageCollection) {
     const collectorsReady = await usageCollection.areAllCollectorsReady();
     const hasUsageCollectors = usageCollection.some(usageCollection.isUsageCollector);
-    if (!collectorsReady) {
+    if (!collectorsReady || this.kibanaStatusGetter === null) {
```

Comment:
```suggestion
    if (!collectorsReady || typeof this.kibanaStatusGetter !== 'function') {
```
Maybe this will be safer in case the api changes?